### ISSUE TITLE
entered unreachable code

### ISSUE THREAD

**Issue by Phill030** [Brendonovich/prisma-client-rust#391]
Executing
```rs
#[derive(serde::Serialize, serde::Deserialize, Debug, PartialEq, Clone, Eq, Hash)]
pub struct Client {
    pub name: String,
    pub hwid: String,
    pub token: String,
    pub first_connection: u64,
    pub last_connection: u64,
}

#[tokio::main]
async fn main() {
    let prisma_client = Arc::new(
        PrismaClient::_builder()
            .build()
            .await
            .expect("Error building prisma client"),
    );

    Db::insert_client(
      prisma_client.clone(),
      Client {
          name: "name".to_string(),
          hwid: "hwid".to_string(),
          token: "token".to_string(),
          first_connection: 0u64,
          last_connection: 0u64,
      },
  )
  .await;
}
```

// database.rs
```rs
async fn insert_client(prisma_client: Arc<PrismaClient>, c: Client) -> () {
        prisma_client
            .clients()
            .create(
                c.hwid,
                c.token,
                c.name,
                c.first_connection.try_into().unwrap(),
                c.last_connection.try_into().unwrap(),
                vec![],
            )
            .exec()
            .await
            .unwrap();
    }
```

results in
```rs
thread 'tokio-runtime-worker' panicked at 'internal error: entered unreachable code', C:\Users\username\.cargo\git\checkouts\prisma-engines-8e1416dd0e571d11\7cc20cc\libs\dml\src\default_value.rs:90:43
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by Brendonovich** [comment]
Can you please provide a small reproduction repo? I don't know what's going on without seeing your schema and figuring out the problem would be easier if I can run something you've already made.

**Comment by Phill030** [comment]
I've created a repository.
(*I'm using replSet's for MongoDB*)

https://github.com/Phill030/prisma-unreachable-code-reproduction

**Comment by Brendonovich** [comment]
Thanks heaps for the reproduction, I think this is an issue with feature gating in the engines. My guess is that since Prisma are used to building all the connectors at once, they always have the engines' `default_generators` feature enabled, but it would seem that they forgot to enable it for the individual mongodb connector.

**Comment by Brendonovich** [comment]
Yep that seems to be the problem, will be fixed in 0.6.10.
